pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -858,6 +859,46 @@ type RetryStrategy struct {
 	Backoff *Backoff `json:"backoff,omitempty" protobuf:"bytes,3,opt,name=backoff,casttype=Backoff"`
 }
 
+// An indicator (i.e. indicative but not accurate) amount of resource * time in seconds, e.g.
+// CPU 1000m * 1m = 1m
+// memory 1Gi * 2m = 2Gi
+// this is represented as duration in seconds, so can be converted to and from duration (with loss of precision below seconds)
+type ResourceUsageIndicator int64
```

Comment:
Seems like all this does is store seconds? Why is it called a `ResourceUsageIndicator`?